src/source/reference/get-feed-empty.rst

```diff
@@ -0,0 +1,17 @@
+.. index:: GET /feed/empty
+.. _get_feed_empty:
+
+GET /feed/emtpy
+================
+
+.. list-table::
+ :widths: 20 80
+
+ * - Scope
+   - <none>
+
+ * - Accept
+   - ``text/xml; charset=UTF-8``
+
+This URL returns an empty file that can be used to temporarily pause the entire ad inventory. It provides a downloadable file with a unique
+name, sufficient to upload it to your xml web server and change the URL to it.
```

Comment:
or, configure that URL in the feed config? Do they _need_ to upload it to their own server?

Reply:
I think the misunderstanding is in the fact that the url is generated on the fly with unique name (uuid) to prevent clashes since we have that restriction on unique url per tenant (or even wider?)